docs/rst/manual/_ext/ert_narratives.py

```diff
@@ -0,0 +1,70 @@
+from sphinx.application import Sphinx
```

Comment:
Why are all of these changes in this PR? 🤔  Think they were already merged in https://github.com/equinor/ert/pull/1543